src/utils/private.ts

```diff
@@ -67,6 +68,10 @@ export interface CommandObj {
   enabled: boolean;
 }
 
+export interface AccessControl {
+  label: boolean;
+}
+
```

Comment:
Please add this to src/types/config.ts

```suggestion
export type AccessControl = Static<typeof AccessControlSchema>;

```